src/Browser.php

```diff
@@ -335,4 +338,33 @@ public function withOptions(array $options)
 
         return $browser;
     }
+
+    /**
+    * Returns the CookieJar handler.
+    * @return CookieJar
+    */
+    public function getCookieJar()
+    {
+        return $this->cookieJar;
+    }
+
+    /**
+    * Set the CookieJar handler.
+    * This object is responsible for managing cookies in all requests.
+    * @param CookieJar $cookieJar
+    */
+    public function setCookieJar($cookieJar) {
```

Comment:
This makes this API mutable, so I would prefer a `withCookieJar()` (or similar) to be in line with the other API methods.

nit: CS